Dockerfile

```diff
@@ -58,9 +58,9 @@ EXPOSE 8080
 # required for external tools to detect this as a go binary
 ENV GOTRACEBACK=all
 
-# KEEP THE FOLLOWING LINES COMMENTED OUT!!! (they will be included within the travis-ci build)
-#build-uncomment ADD docker/MANIFEST /
-#build-uncomment COPY docker/entrypoint.sh /
+# KEEP THE FOLLOWING LINES COMMENTED OUT!!! (they will be included within the CI build)
```

Comment:
this sounds like there's magic involved, can you explain this?

Reply:
The magic is called `sed` ;)
It was implemented directly in .travis-ci.yml / GH CI workflow, but it has now moved to keptn/gh-action-build-docker-image: https://github.com/keptn/gh-action-build-docker-image/blob/d0d6abbe64be2deafc39f33322c1768fc5a286ab/build_docker_image.sh#L32-L34

The idea is: You don't need it to build the docker image, but when we want an official Keptn Docker Image, we add it